pkg/cassandra/config/config.go

```diff
@@ -31,6 +31,7 @@ type Configuration struct {
 	Keyspace             string        `validate:"nonzero"`
 	ConnectionsPerHost   int           `validate:"min=1" yaml:"connections_per_host"`
 	Timeout              time.Duration `validate:"min=500"`
+	ReconnectInterval    time.Duration `validate:"min=500" yaml:"reconnect_interval"`
```

Comment:
I'm not convinced that the minimum of 500ns is a good value. Could you elaborate on how this was chosen?

(I think we need to revisit the `Timeout` as well separately)

Reply:
I'm not sure that I really understand the `validate:"min=500"` section. It seems that it wasn't actually enforced in my tests. (As in, I would not pass any value, resulting in 0, which is less than 500 and it would pass validation. Unless that's a special case?)

I copied `Timeout` value for this.

Reply:
Hmmm, it looks like these validations would only be applied if this configuration file was validated with something like https://github.com/go-validator/validator

I'll make a separate ticket to address this